hasura/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hasura/graphql-engine:v1.3.3.cli-migrations-v2
+FROM fedormelexin/graphql-engine-arm64:v1.3.3.cli-migrations-v2
```

Comment:
Should this have been committed? Isn't it going to break most people's setups?

Reply:
you're right, this slipped through. thanks